### PR TITLE
README.md: Add more/clean up links to consuming projects

### PR DESCRIPTION
**Pull request by cgwalters**
Add meta-updater and QtOTA, and delete OpenEmbedded since it's implied by the
first two. Merge rpm-ostree + Atomic Host since they're close. Clarify
gnome-continuous a bit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,26 +38,30 @@ use OSTree as a "deduplicating hardlink store".
...
+
+[rpm-ostree](https://github.com/projectatomic/rpm-ostree) is a next-generation
+hybrid package/image system for [Fedora](https://getfedora.org/) and [CentOS](https://www.centos.org/),
+used by the [Atomic Host](http://www.projectatomic.io/) project.
+By default it uses libostree to atomically replicate a base OS (all dependency
+resolution is done on the server), but it supports "package layering", where
+additional RPMs can be layered on top of the base.  This brings a "best of both worlds""
+model for image and package systems.
+
+[flatpak](https://github.com/alexlarsson/xdg-app) uses libostree for desktop
```
Comment (jlebon) [inline README.md]: This is at https://github.com/flatpak/flatpak now.

**Comment by cgwalters** [comment]
Fixup ⬆️ also with link to EndlessOS.

**Comment by jlebon** [comment]
@rh-atomic-bot r+ b43c736

**Comment by jlebon** [comment]
Unrelated, but thought I'd link it here: https://github.com/jlebon/redhat-ci/issues/27. It's really silly to be running all these tests (we've spawned 12 suite runs so far from this PR) when we're just updating the README.

**Pull request closed without merging**